stories/core/ExpandableBlock.stories.tsx

```diff
@@ -140,3 +140,16 @@ Small.args = {
   title: 'Basic Block',
   size: 'small',
 };
+
+export const Borderless: Story<Partial<ExpandableBlockProps>> = (args) => {
```

Comment:
I noticed that there is no borderless story for `Button`, so I'm not sure if this story is needed either, but I figured it couldn't hurt

Reply:
That's because borderless button should be very careful used (almost discouraged). It is better for icons than for text buttons

Reply:
I assumed the expandable block was a type of text button more so than an icon.

The stories have been added back.